src/compiler-team.md

```diff
@@ -0,0 +1,106 @@
+(UTC-4 typically, but daylight savings time sometimes makes things
+complicated).
+
+The meeting is held over a "chat medium" -- it used to be IRC, but we
+are currently in the process of evaluating other alternatives. Check
+[the etherpad] to find the current home (and see
+[this internals thread][thread] for some ongoing discussion).
+
+[etherpad]: https://public.etherpad-mozilla.org/p/rust-compiler-meeting
+[thread]: https://internals.rust-lang.org/t/where-should-the-compiler-team-and-perhaps-working-groups-chat/7894
+[procedure]: https://forge.rust-lang.org/rustc-bug-fix-procedure.html
+
+## Team membership
+
+Membership in the Rust team is typically offered when someone has been
+making significant contributions to the compiler for some
+time. Membership is both a recognition but also an obligation:
+compiler team members are generally expected to help with upkeep as
+well as doing reviews and other work.
+
+If you are interested in becoming a compiler team member, the first
+thing to do is to start fixing some bugs, or get involved in a working
+group.
+
+### r+ rights
+
+Once you have made a number of individual PRs to rustc, we will often
+offer r+ privileges. This means that you have the right to instruct
+"bors" (the robot that manages which PRs get landed into rustc) to
+merge a PR
+([here are some instructions for how to talk to bors][homu-guide]).
+
+[homu-guide]: https://buildbot2.rust-lang.org/homu/
+
+The guidelines for reviewers are as follows:
+
+- You are always welcome to review any PR, regardless of who it is
+  assigned to.  However, do not r+ PRs unless:
+  - You are confident in that part of the code.
+  - You are confident that nobody else wants to review it first.
+    - For example, sometimes people will express a desire to review a
+      PR before it lands, perhaps because it touches a particularly
+      sensitive part of the code.
+- Always be polite when reviewing: you are a representative of the
+  Rust project, so it is expected that you will go above and beyond
+  when it comes to the [Code of Conduct].
+
+[Code of Conduct]: https://www.rust-lang.org/en-US/conduct.html
+
+### high-five
+
+Once you have r+ rights, you can also be added to the high-five
+rotation. high-five is the bot that assigns incoming PRs to
+reviewers. If you are added, you will be randomly selected to review
+PRs. If you find you are assigned a PR that you don't feel comfortable
+reviewing, you can also leave a comment like `r? @so-and-so` to assign
+to someone else -- if you don't know who to request, just write `r?
+@nikomatsakis for reassignment` and @nikomatsakis will pick someone
+for you.
+
+[hi5]: https://github.com/rust-highfive
+
+Getting on the high-five list is much appreciated as it lowers the
+review burden for all of us! However, if you don't have time to give
+people timely feedback on their PRs, it may be better that you don't
+get on the list.
+
+### Team membership
```

Comment:
Perhaps rename to `Full Team Membership`? There is already a section entitled "Team membership".